com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/JavaFileJDT.java

```diff
@@ -592,4 +594,86 @@ protected File getFile() {
 		return null;
 	}
 
+	private boolean typeMatch(String expectType, String paramType) {
+		boolean match = false;
+
+		if (expectType.equals(paramType)) {
+			match = true;
+		}
+		else if (paramType.equals("null")) {
+			match = true;
+		}
+		else if (expectType.contains(".") && !paramType.contains(".") && expectType.endsWith(paramType)) {
```

Comment:
For your case "java.lang.Integer" and "my.test.Integer", it wouldn't go into this condition, this one and the below one are expected to handle the cased like expectedType == "java.lang.Integer" and paramType = "Integer" something without including "."